src/integration/utils/runTests/constants/services.js

```diff
@@ -0,0 +1,342 @@
+    // PGL: '/indonesia/indonesia-41635759',
+  },
+
+  japanese: {
+    frontPage: '/japanese',
+    articles: '/japanese/articles/cdd6p3r9g7jo',
+    mediaAssetPage: '/japanese/video-23248670',
+    // PGL: '/japanese/features-and-analysis-42786589',
+  },
+
+  korean: {
+    frontPage: '/korean',
+    articles: '/korean/articles/c3mn1lvz65xo',
+    liveRadio: '/korean/bbc_korean_radio/liveradio',
+    mediaAssetPage: '/korean/media-23248686',
+    // PGL: '/korean/features-41397333',
+  },
+
+  kyrgyz: {
+    frontPage: '/kyrgyz',
+    articles: '/kyrgyz/articles/c3xd4xg3rm9o',
+    liveRadio: '/kyrgyz/bbc_kyrgyz_radio/liveradio',
+    mediaAssetPage: '/kyrgyz/media-23257484',
+    // PGL: '/kyrgyz/world-40847556',
+  },
+
+  marathi: {
+    frontPage: '/marathi',
+    articles: '/marathi/articles/cp47g4myxz7o',
+    mediaAssetPage: '/marathi/media-23127353',
+    // PGL: '/marathi/india-42894522',
+  },
+
+  mundo: {
+    frontPage: '/mundo',
+    articles: '/mundo/articles/ce42wzqr2mko',
+    // PGL: '/mundo/deportes-36935058',
+  },
+
+  naidheachdan: {
+    articles: '/naidheachdan/sgeulachdan/c18q7nedn2ko',
+  },
+
+  nepali: {
+    frontPage: '/nepali',
+    articles: '/nepali/articles/cl90j9m3mn6o',
+    liveRadio: '/nepali/bbc_nepali_radio/liveradio',
+    mediaAssetPage: '/nepali/media-23269034',
+    // PGL: '/nepali/news-50627370',
+  },
+
+  news: {
+    articles: '/news/articles/cn7k01xp8kxo',
+  },
+
+  pashto: {
+    frontPage: '/pashto',
+    articles: '/pashto/articles/cyjmdl92z3ro',
+    liveRadio: '/pashto/bbc_pashto_radio/liveradio',
+    mediaAssetPage: '/pashto/media-23257523',
+    // PGL: '/pashto/arts-and-literature-50230813',
+  },
+
+  persian: {
+    frontPage: '/persian',
+    articles: '/persian/articles/cej3lzd5e0go',
+    liveRadio: '/persian/bbc_persian_radio/liveradio',
+    mediaAssetPage: '/persian/iran-23231114',
+    // PGL: '/persian/magazine-49281981',
+  },
+
+  pidgin: {
+    frontPage: '/pidgin',
+    articles: '/pidgin/articles/cwl08rd38l6o',
+    mediaAssetPage: '/pidgin/23248703',
+    // PGL: '/pidgin/sport-23252855',
+  },
+
+  portuguese: {
+    frontPage: '/portuguese',
+    articles: '/portuguese/articles/cd61pm8gzmpo',
+    // PGL: '/portuguese/geral-40302633',
+  },
+
+  punjabi: {
+    frontPage: '/punjabi',
+    articles: '/punjabi/articles/c0l79lr39qyo',
+    mediaAssetPage: '/punjabi/media-23248705',
+    // PGL: '/punjabi/india-42928885',
+  },
+
+  russian: {
+    frontPage: '/russian',
+    articles: '/russian/articles/ck7pz7re3zgo',
+    // PGL: '/russian/features-45782775',
+  },
+
+  scotland: {
+    articles: '/scotland/articles/czwj5l0n210o',
+  },
+
+  serbian: {
+    variants: {
+      cyr: {
+        frontPage: '/serbian/cyr',
+        articles: '/serbian/articles/c805k05kr73o/cyr',
+      },
+      // 404ing
+      // lat: {
+      //   frontPage: '/serbian/lat',
+      //   articles: '/serbian/articles/c805k05kr73o/lat',
+      // PGL: '/serbian/srbija-46748932/lat',
```

Comment:
This PGL has the wrong URL, if you update this to `serbian/lat/srbija-46748932` then the PGL tests should work